service/src/test/java/bio/terra/landingzone/library/landingzones/definition/factories/CromwellBaseResourcesFactoryTest.java

```diff
@@ -38,60 +36,44 @@ void setUp() {
   }
 
   @Test
-  void deploysLandingZoneV1_resourcesAreCreated() throws InterruptedException {
+  void deploysLandingZoneV1_resourcesAreCreatedAndDeleted()
```

Comment:
I combined create and delete tests into 1 for 2 reasons, save time (about 8m - only have to deploy LZ once) and prevent test interference because they share the same resource group.